linchpin/__init__.py

```diff
@@ -1026,6 +1027,8 @@ def _invoke_playbooks(self, resources={}, action='up', console=True,
             self.set_evar('state', 'absent')
 
         inventory_src = '{0}/localhost'.format(self.workspace)
+        if not hasattr(self, 'pbar'):
```

Comment:
Why don't we just only use `setup_pbar()` within the `LinchpinAPI` class instead of having `linchpin/cli/__init__.py` also run the code?

Reply:
@14rcole I didn't get you. 
Currently self.setup_pbar is only used in linchpin/__init__ and shell/__init__.py 
I dont see it being run in cli implementation. 
Could you clarify on it.

Reply:
Sorry, I meant `shell/__init__.py`.  Basically, there's no reason for a class outside of `linchpin/__init__.py` to have to setup the progress bar, so if we just run `setup_pbar()` on line 1034 here we can remove the if statement on line 1033